contracts/wrapper/NameWrapper.sol

```diff
@@ -942,29 +964,28 @@ contract NameWrapper is
         string memory label,
         address wrappedOwner,
         uint32 fuses,
-        uint64 expiry,
         address resolver
     ) private returns (uint64) {
         // Mint a new ERC1155 token with fuses
         // Set PARENT_CANNOT_REPLACE to reflect wrapper + registrar control over the 2LD
         bytes32 labelhash = keccak256(bytes(label));
         bytes32 node = _makeNode(ETH_NODE, labelhash);
-        uint32 oldFuses;
+        _checkForParentControlledFuses(node, fuses);
 
-        (, oldFuses, expiry) = _getETH2LDDataAndNormaliseExpiry(
```

Comment:
Please remove `function _getETH2LDDataAndNormaliseExpiry` on line 924 which is no longer in use by anyone (and probably bunch of tests as well

Reply:
Good catch, will remove and check tests for normalisation of .eth names